inc/Ui/Search.php

```diff
@@ -579,33 +581,37 @@ protected function getFulltextResultsHTML($data, $highlight)
             }
 
             $snippet = '';
-            $lastMod = '';
+            $hits = '';
+            $resultBody = [];
             $mtime = filemtime(wikiFN($id));
+            $lastMod = '<span class="lastmod">' . $lang['lastmod'] . '</span> ';
```

Comment:
Why is all of this outside of the `if ($cnt !== 0)`? Should anything be displayed if the count is 0?

Reply:
There are circumstances where `$cnt === 0` but the page still matches the search. (Right now, I don't recall what causes them :/)
In these cases we display the pagename, but no hits and no snippet. I think it would make sense to at least show the last mod date as well?

Reply:
This might have to do with plugins that add additional content to the search (e.g. the discussion plugin does this) - though I think the discussion plugin should at least also provide snippets but I haven't checked.